topcoffea/cfg/mc_signal_samples.cfg

```diff
@@ -84,10 +84,25 @@
 
 # Central UL17 samples
 root://ndcms.crc.nd.edu/
-../../topcoffea/json/signal_samples/central_UL/UL17_ttHnobb.json
-../../topcoffea/json/signal_samples/central_UL/UL17_TTWJetsToLNu.json
-../../topcoffea/json/signal_samples/central_UL/UL17_TTZToLLNuNu_M_10.json
-../../topcoffea/json/signal_samples/central_UL/UL17_tZq.json
+../../topcoffea/json/background_samples/central_UL/UL17_WZTo3LNu.json
+../../topcoffea/json/background_samples/central_UL/UL17_WWTo2L2Nu.json
+../../topcoffea/json/background_samples/central_UL/UL17_ZZ.json
+../../topcoffea/json/background_samples/central_UL/UL17_WWW_4F.json
+../../topcoffea/json/background_samples/central_UL/UL17_WWZ_4F.json
+../../topcoffea/json/background_samples/central_UL/UL17_WZZ_ext.json
+../../topcoffea/json/background_samples/central_UL/UL17_ZZZ.json
+../../topcoffea/json/background_samples/central_UL/UL17_TTGJets.json
+../../topcoffea/json/background_samples/central_UL/UL17_WZZ.json
+#../../topcoffea/json/background_samples/central_UL/UL17_WJetsToLNu.json
+#../../topcoffea/json/background_samples/central_UL/UL17_TTZToLLNuNu.json
+#../../topcoffea/json/background_samples/central_UL/UL17_TTWJetsToLNu.json
+#../../topcoffea/json/background_samples/central_UL/UL17_TTJets.json
+#../../topcoffea/json/background_samples/central_UL/UL17_t.json
+#../../topcoffea/json/background_samples/central_UL/UL17_tW.json
+#../../topcoffea/json/background_samples/central_UL/UL17_tbar.json
+#../../topcoffea/json/background_samples/central_UL/UL17_tbarW.json
+#../../topcoffea/json/background_samples/central_UL/UL17_DY10to50.json
+#../../topcoffea/json/background_samples/central_UL/UL17_DY50.json
```

Comment:
This are all the backgrounds that correspond to the control regions in TOP-19-001. I've left the commented out so they don't pollute the datacards.